README.md

```diff
@@ -57,6 +57,7 @@ A collection of Android security-related resources.
 
 ### Static Analysis Tools
 
+1. [APKdevastate](https://github.com/rafigk2v9c/APKdevastate/) - Advanced analysis software for APK payloads created by RATs.
```

Comment:
Please add it below the last active entry, that is, `[Detekt](https://github.com/detekt/detekt)`